### PR TITLE
Introduce rule for Null Coalesce Equal Operator

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: tests/input/new_with_parentheses.php [hunk 1]
```diff
@@ -25,4 +25,4 @@
 $z = new stdClass ? new stdClass : new stdClass;
 
 $q = $q ?: new stdClass;
-$e = $e ?? new stdClass;
+$e = $d ?? new stdClass;
```
Comment (carusogabriel) [inline tests/input/new_with_parentheses.php]: I've changed this one to avoid having these unit tests affected 👍

**Comment by carusogabriel** [comment]
@greg0ire Yeah, I know, the `CODEOWNERS` got trigged and I couldn't remove the review, my bad.

**Comment by greg0ire** [comment]
Oh right, not sure what you can do against that… maybe this is not triggered when using a draft PR? But I'm not 100 sure github actions are triggered then.

File: tests/input/null_coalesce_operator.php [hunk 2]
```diff
@@ -4,8 +4,6 @@
 
 $foo = isset($_GET['foo']) ? $_GET['foo'] : 'foo';
 
-$bar = isset($bar) ? $bar : 'bar';
-
```
Comment (carusogabriel) [inline tests/input/null_coalesce_operator.php]: Moved to `tests/input/null_coalesce_equal_operator.php` as the fixed output now contains an `??=`.

**Comment by carusogabriel** [comment]
@lcobucci Can you help me here please with `make update-compatibility-patch`? I can't figure out what I'm doing wrong 😕

**Comment by greg0ire** [comment]
> But I'm not 100% sure github actions are triggered then.

Looks like they are, actually: https://github.com/sonata-project/SonataBlockBundle/pull/694

**Pull request closed without merging**